reactor-core-micrometer/src/main/java/reactor/core/observability/micrometer/MicrometerObservationListener.java

```diff
@@ -19,13 +19,16 @@
 import io.micrometer.observation.Observation;
 
 import reactor.core.observability.SignalListener;
+import reactor.core.observability.micrometer.DocumentedObservationListenerTags.ObservationTags;
 import reactor.core.publisher.SignalType;
 import reactor.util.Logger;
 import reactor.util.Loggers;
 import reactor.util.annotation.Nullable;
 import reactor.util.context.Context;
 import reactor.util.context.ContextView;
 
+import static reactor.core.observability.micrometer.DocumentedObservationListenerTags.ObservationTags.*;
```

Comment:
Consider explicit listing of static imports here.

Reply:
ah I missed that one in my recent commit, 1sec